public/editor/scripts/editor/js/bramble-editor.js

```diff
@@ -14,7 +14,13 @@ define(function(require) {
       // Start loading the Bramble editor resources
       Bramble.load("#webmaker-bramble",{
         url: options.editorUrl,
-        hideUntilReady: true
+        hideUntilReady: true,
+        extensions: {
+          enable: [
+            // Watch and warn users on rename or delete of index.html
+            'bramble-watch-index.html'
```

Comment:
So it sounds like we need a patch to the Bramble client code (i.e., where you're pointing here) to take `options.url` and parse it, then merge any search params we find there with whatever options we want to pass down (i.e., whatever we got on `options`).  Want me to do that?